test/dialect/oracle/test_reflection.py

```diff
@@ -630,13 +630,7 @@ def all_tables_compression_missing():
 
 
 def all_tables_compress_for_missing():
-    with testing.db.connect() as conn:
```

Comment:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4829